f600233d1b154d507b782f84604bcb12/update/sys_script_include_b29002b097a211106fa8b4b3f153afc4.xml

```diff
@@ -427,21 +367,59 @@ Igce.prototype = {
             }
         }
 
-        // Get Surge Capabilities
-        var surgeCapabilities
-        if (requirementsCostEstimate) {
-            var gr = new GlideRecord('x_g_dis_atat_requirements_cost_estimate');
-            gr.get(requirementsCostEstimate);
-            surgeCapabilities = parseInt(gr.surge_capabilities.toString());
+        // Get instructions
+        var igceInstructions = this.getIgceInstructions(this.acquisitionPackage.sys_id);
+        var igceSurgeCapabilities = null;
+        var instructions = null;
+
+        if (igceInstructions) {
+            if (igceInstructions.surge_requirement_capabilities === "YES") {
+                igceSurgeCapabilities = igceInstructions.surge_requirement_capacity;
+            }
+            var toolsUsed = "";
+            if (igceInstructions.how_est_dev_tools_used) {
+                toolsUsed += igceInstructions.how_est_dev_tools_used;
+            }
+            if (igceInstructions.how_est_dev_other_tools_used) {
+                toolsUsed += " " + igceInstructions.how_est_dev_other_tools_used;
+            }
+
+            instructions = {
+                estimateDescription: igceInstructions.how_est_dev_cost_estimate_description,
+                assumptionsMade: "assumptions",
+                toolsUsed: toolsUsed,
+                informationSource: "info source",
+                previousEstimateComparison: igceInstructions.how_est_dev_prev_cost_estimate_comp_option
+            };
+        }
+
+        // Get Contracting Shop
+        var cShop;
+        if (acq_package.contracting_shop) {
+            cShop = acq_package.contracting_shop.toString();
         }
 
+
+        // Get all IGCE Estimates that have the acquisition package as the FK
+        var igce = this.getIgceEstimates(this.acquisitionPackage.sys_id);
+        // Determine selected_periods for instance_info and classification_level
+        this.getIgceProperties(igce);
+        // Map quantity to each IGCE with selected period
+        this.mapQuantity(igce)
+        // Map igce estimates to periods estimate
+        this.mapIgceToPeriodsEstimate(igce, periodsEstimate);
+
         // IGCE generated-document payload
+
         var igce_payload = {
             documentType: "INDEPENDENT_GOVERNMENT_COST_ESTIMATE",
             templatePayload: {
                 fundingDocument: fundingDocument,
-                surgeCapabilities: surgeCapabilities,
                 periodsEstimate: periodsEstimate,
+                contractingShop: cShop,
+                instructions: instructions,
+                surgeCapabilities: igceSurgeCapabilities,
+                // igce_test: igce
```

Comment:
Ready to remove?

Reply:
https://github.com/dod-ccpo/atat-snow/pull/249#discussion_r1061874622